app/js/main.js

```diff
@@ -79,6 +89,32 @@ fileDrop.addEventListener(DRAG_DROP_EVENT, function (e) {
             dragSource = null;
         });
 
+        tr.addEventListener(TOUCH_START_EVENT, () => {
+            tr.classList.add(DRAG_OVER_CSS);
+            clearButton.classList.add(DRAG_OVER_CSS);
+            dragState = true;
+            dragSource = tr;
+        });
+
+        tr.addEventListener(TOUCH_END_EVENT, () => {
+            tr.classList.remove(DRAG_OVER_CSS);
+            clearButton.classList.remove(DRAG_OVER_CSS);
+            dragState = false;
+        });
+
+        tr.addEventListener(TOUCH_MOVE_EVENT, (e) => {
+            e.preventDefault();
+            if (dragState) {
+                const touch = e.touches[0];
+                const element = document.elementFromPoint(touch.clientX, touch.clientY);
+                if (element && element.tagName === TABLE_ROW_TAG) {
```

Comment:
can be simplified to optional chaining `(element?.tagName === TABLE_ROW_TAG)`

Reply:
Please change

Reply:
Fixed with [0438a2b](https://github.com/markuszeller/image-stitcher/pull/14/commits/0438a2b9bcc69846dafd3564f741d14469693762)